packages/cli/src/lib/implementation/compare.options.ts

```diff
@@ -0,0 +1,20 @@
+import { Options } from 'yargs';
+import type { CompareOptions } from './compare.model';
+
+export function yargsCompareOptionsDefinition(): Record<
+  keyof CompareOptions,
+  Options
+> {
+  return {
+    before: {
+      describe: 'Path to source report.json',
+      type: 'string',
+      demandOption: true,
```

Comment:
In the future we can use prompts and move this line into a middleware with nice user prompts